scipy/integrate/_bvp.py

```diff
@@ -1078,12 +1085,16 @@ def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
 
         col_res, y_middle, f, f_middle = collocation_fun(fun_wrapped, y,
                                                          p, x, h)
+        bc_res = bc_wrapped(y[:, 0], y[:, -1], p)
+        max_bc_res = np.max(abs(bc_res)) * tol / tol_bc
```

Comment:
I suggest we define simply `max_bc_res  = np.max(np.abs(bc_res))` (see below).

Reply:
That change has been made.